source/read-from-mongodb.txt

```diff
@@ -42,6 +42,18 @@ Overview
 
          .. include:: /scala/filters.txt
 
+.. important:: Change Stream Schema Inference
+
+   When the {+driver-short+} infers the schema of a data frame 
+   read from a change stream, by default,
+   it will use the schema of the underlying collection rather than the schema
```

Comment:
S: Remove the "by default" here. It is implicit as the next sentence explains how to change the behavior

Reply:
I see what you mean, but to me it benefits from some indication that this behavior can be changed (apart from the following sentence, I mean). Especially when this isn't the same page where this option is documented, so the reader might not be in the mindset of settings they can turn on or off.